setup.cfg

```diff
@@ -0,0 +1,3 @@
+[pycodestyle]
+exclude=newdle/client,__pycache__
+max-line-length=89
```

Comment:
Aren't we using 120?

Reply:
no currently we're on the 89 default from black